charts/atlantis/templates/servicemonitor.yaml

```
@@ -5,6 +5,9 @@ metadata:
   name: {{ template "atlantis.fullname" . }}
   labels:
 {{- include "atlantis.labels" . | nindent 4 }}
+  {{- with .Values.servicemonitor.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Easier to understand if it's aligned 😃:
```suggestion
    {{- include "atlantis.labels" . | nindent 4 }}
    {{- with .Values.servicemonitor.additionalLabels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```